sources/Renderer/RenderSystem.cpp

```diff
@@ -26,6 +26,7 @@
 #include <unordered_map>
 
 #include "../Core/PrintfUtils.h"
+#include "TextureUtils.h"
```

Comment:
If I haven't missed anything, I believe this and the two _TextureUtils.*_ files don't need to be changed anymore.